### DISCUSSION TITLE
About lyric translation

### DISCUSSION THREAD

**Discussion by DuskRa1n** [ciderapp/Cider#1363] [Feature Request]
I am a Chinese user. I can read both simplified Chinese and traditional Chinese. However if I choose the prefer language from them such as simplified one, when it find a lyric of traditional one, it will translate it to simplified one. In my eyes, both of the two lyrics are the same. It really makes my experience bad. So I have two ideas.
1. Set a switch button of lyric translation
2. Set a option of Chinese(simplified and traditional)

Maybe you have a better solution, I sincerely hope the issue will be solved.
Best wishes.

**Comment by maikirakiwi** [comment]
Sorry couldn’t understand what you were trying to say.
说实话我没看懂。
要不您用中文跟我再解释一遍？

**Reply by DuskRa1n** [reply]
我英文有这么烂吗。。。。就是我如果选择了歌词语言偏好，遇到非这个偏好语言的歌词它会给出翻译，我如果偏好选的是简中，那么它在找到繁中歌词之后给我一个简中的翻译，这样就有点尴尬，我想了两个解决办法，一个是提供一个歌词翻译的开关，另一个是在偏好歌词里加一个中文（包括简中和繁中）

**Reply by DuskRa1n** [reply]
<img width="1024" alt="Snipaste_2022-08-15_19-33-01" src="https://user-images.githubusercontent.com/88879127/184628822-02b648c8-fbb5-412d-8454-07f27bcc3a49.png">
<img width="1024" alt="Snipaste_2022-08-15_19-33-12" src="https://user-images.githubusercontent.com/88879127/184628826-b57413b6-b265-4db0-9cf9-6f39bc1363f6.png">

**Comment by maikirakiwi** [comment]
还有我们有官方Q群的531185058

**Comment by jay900604** [comment]
這個問題基本上 Cider 應該是無法解決的，因為這個是歌詞庫的問題，繁體中文語言的歌詞是比較不完整的，因此當你選擇設定繁體中文，但還是出現簡體中文語言的歌詞是屬於正常的

**Reply by DuskRa1n** [reply]
这个我理解，我想的是能不能推出一个翻译开关，我不需要的时候就把翻译关了，需要的时候再开起来
